test/TEST-83-BUS-POLKIT/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
just merge this in TEST-74-AUX-UTILS no need for a separate one

Reply:
Yeah. I created a separate test, because I couldn't decide to which existing one to place it. (And also because it currently builds a separate image, but I've already realized that's not really necessary.)